### DISCUSSION TITLE
How to change the prompt words of the search box to Chinese

### DISCUSSION THREAD

**Discussion by w1ndys** [squidfunk/mkdocs-material#6862] [Q&A] [answered]
I searched for information, and one of the methods was to write "lang: zh" in the configuration parameters of the search plug-in, but I didn't get any correct results. After asking ChatGPT, the method he gave was to directly modify the site-package template. file, but this is too intrusive and not conducive to my deployment in Netlify, so I'm wondering if there is any other non-intrusive solution

such  as
![image](https://github.com/squidfunk/mkdocs-material/assets/109416673/38c70fb1-4e59-4d6b-aa9b-ea5bbc79e3f1)

**Comment by alexvoss** [comment, marked as answer]
The relevant setting should be this:

```yaml
theme:
  name: material
  language: zh
```

**Reply by w1ndys** [reply]
This is awesome, turns out I needed to change "lang" to "language", thank you very much for your help, I guess I should be ashamed of myself for having this problem

**Reply by alexvoss** [reply]
No worries. In fact, we are keen to see how people search of answers so that we can improve the documentation. I might suggest a change to the wording in the configuration to make sure people who look at search see that they might only need to change the setting for the theme as a whole - which is likely to be the more frequent use case. The current wording might not make that clear enough.
